lib/nvme/nvme_ctrlr.c

```diff
@@ -731,6 +732,15 @@ nvme_ctrlr_process_init(struct spdk_nvme_ctrlr *ctrlr)
 	uint32_t ready_timeout_in_ms;
 	int rc;
 
+	/*
+	 * May need to avoid accessing any register on the target controller
+	 * for a while. Return early without touching the FSM.
+	 */
+	if (ctrlr->sleep_timeout_in_ms && 
+			(spdk_get_ticks() <= ctrlr->sleep_timeout_in_ms)) {
```

Comment:
It looks like the sleep timeout is actually in ticks, not milliseconds, so please name this field sleep_timeout_ticks or similar.